example/AsProject/Physical/ARM/X20CP0410/Cpu.sw

```diff
@@ -92,8 +92,9 @@
     <LibraryObject Name="Arnc0man" Source="Libraries.Motion.Arnc0man.lby" Memory="UserROM" Language="binary" Debugging="true" />
     <LibraryObject Name="Arnc0mx" Source="Libraries.Motion.Arnc0mx.lby" Memory="UserROM" Language="binary" Debugging="true" />
     <LibraryObject Name="TRF_LIB" Source="Libraries.Motion.TRF_LIB.lby" Memory="UserROM" Language="binary" Debugging="true" />
-    <LibraryObject Name="chopper" Source="Libraries.Loupe.chopper.lby" Memory="UserROM" Language="ANSIC" Debugging="true" />
+    <LibraryObject Name="chopper" Source="Libraries.Loupe.chopper.lby" Memory="UserROM" Language="ANSIC" AdditionalLibraryDirectories="\..\..\src\Ar\Chopper\lib\SG4\Arm\" AdditionalLibraries="AR" Debugging="false" />
```

Comment:
I dont know if you actually need this if the library is in the folder

Reply:
AR? I assume it does not need to be in the Chopper folder but I wasnt sure where else it should live, any other recommendations?